packages/web/.env

```diff
@@ -11,6 +11,7 @@ NEXT_PUBLIC_TFM_API_BASE_URL=https://api.tfm.com
 SENTRY_IGNORE_API_RESOLUTION_ERROR=1
 NEXT_PUBLIC_SQUID_INTEGRATOR_ID=osmosis-api
 NEXT_PUBLIC_SIDECAR_BASE_URL=https://sqs.osmosis.zone
+BLOCKAID_BASE_URL=http://api.blockaid.io:80
```

Comment:
Actually it is already defined on line 50 on .env file

Reply:
Ah it just needs to be uncommented 👍